src/_data/cloud-error-messages.yml

```diff
@@ -0,0 +1,86 @@
+# Error messages
```

Comment:
The PR didn't explain, but is this file generated or copied?

Reply:
The Suggested actions are generated from a YAML file sourced from info provided by devs.  Devs are planning to create the YAML for the error messages in the next update (after this ece-tools release goes out).